tests/Unit/Ads/AccountServiceTest.php

```diff
@@ -159,8 +162,14 @@ public function test_get_connected_account() {
 			->willReturn( self::TEST_ACCOUNT_ID );
 
 		$this->options->method( 'get' )
-			->with( OptionsInterface::ADS_ACCOUNT_CURRENCY )
-			->willReturn( self::TEST_CURRENCY );
+			->withConsecutive(
+				[ OptionsInterface::ADS_ACCOUNT_OCID ],
+				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ]
+			)
+			->willReturnOnConsecutiveCalls(
+				self::TEST_ACCOUNT_OCID,
+				self::TEST_CURRENCY
+			);
```

Comment:
@asvinb the `OptionsInterface::ADS_ACCOUNT_CURRENCY` is called twice in the [`get_connected_account`](https://github.com/woocommerce/google-listings-and-ads/blob/57579940eca4ef7645c795de0ae59ea741bedc88/src/Ads/AccountService.php#L75) method, once for the currency and a second time for the symbol. We just need to update the tests to include the second call.

```suggestion
			->withConsecutive(
				[ OptionsInterface::ADS_ACCOUNT_OCID ],
				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ],
				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ]
			)
			->willReturnOnConsecutiveCalls(
				self::TEST_ACCOUNT_OCID,
				self::TEST_CURRENCY,
				self::TEST_CURRENCY
			);
```